tests/test_matrix_utils_sync_bootstrap_login.py

```
@@ -680,6 +685,7 @@ async def test_login_matrix_bot_e2ee_store_path_creation(
 
     assert result is True
     mock_makedirs.assert_any_call(TEST_E2EE_STORE_PATH, exist_ok=True)
+    ensure_cross_signed.assert_awaited_once_with(mock_client, password="password")
```

Ruff (0.15.21)

[error] Possible hardcoded password assigned to argument: "password"

(S106)